pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools"]
+
+[project]
+name = "package-metrics"
+version = "0.1.0"
+readme = "README.md"
+description = "Python tool to track outdated dependencies."
+authors = [
+    {name = "Dimagi", email = "dev@dimagi.com"},
```

Comment:
I think my internal syntax checker just got a kidney stone. Is this TOML syntax? Not compatible with JSON, not even YAML! Why does there need to be so many different variations?

/rant